adapters/adoppler/adoppler.go

```diff
@@ -28,11 +32,13 @@ type adsImpExt struct {
 }
 
 type AdopplerAdapter struct {
-	endpoint string
+	endpoint *template.Template
 }
 
 func NewAdopplerBidder(endpoint string) *AdopplerAdapter {
-	return &AdopplerAdapter{endpoint}
+	e := endpoint + "/processHeaderBid/{{.AdUnit}}"
+
+	return &AdopplerAdapter{template.Must(template.New("endpoint").Parse(e))}
```

Comment:
Please don't use `Must` in parsing the endpoint. It's better to log a descriptive message instead of causing a panic.  Here's an example from the adhese adapter:
```
func NewAdheseBidder(uri string) *AdheseAdapter {
	template, err := template.New("endpointTemplate").Parse(uri)
	if err != nil {
		glog.Fatal("Unable to parse endpoint url template")
		return nil
	}
	return &AdheseAdapter{endpointTemplate: *template}
}
```